src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -336,7 +336,7 @@ private async Task FormatDocumentCreatedFromTemplateAsync(IVsHierarchy hierarchy
             var formattedText = formattedRoot.GetText(unformattedText.Encoding, unformattedText.ChecksumAlgorithm);
 
             // Ensure the line endings are normalized. The formatter doesn't touch everything if it doesn't need to.
-            var targetLineEnding = formattingOptions.Options.GetOption(FormattingOptions.NewLine)!;
+            var targetLineEnding = formattingOptions.GetOption(FormattingOptions2.NewLine)!;
```

Comment:
belagh.  we have `2` options here as well :(

Reply:
yeah, actually `-2` options are the ones we should be using everywhere except for public API at this point. I hope to eliminate but not sure yet.